### PR TITLE
feat: Multiple accounts

### PR DESCRIPTION
**Pull request by gregor**

### CODE REVIEW

File: app/script/user/UserRepository.js [hunk 1]
```diff
@@ -946,4 +957,17 @@ z.user.UserRepository = class UserRepository {
         });
       });
   }
+
+  map_guest_status(user_ets = this.users()) {
+    const team_members = this.team_members();
+
+    console.log('user_ets', user_ets);
```
Comment (herrmannplatz) [inline app/script/user/UserRepository.js]: logs

File: test/unit_tests/team/TeamRepositorySpec.js [hunk 2]
```diff
@@ -82,19 +76,6 @@ describe('z.team.TeamRepository', () => {
...
-          expect(entity.creator).toEqual(team_metadata.creator);
-          expect(entity.id).toEqual(team_metadata.id);
-          expect(entity.name()).toEqual(team_metadata.name);
-          done();
-        })
-        .catch(done.fail);
-    });
-  });
-
   xdescribe('get_team_members()', () => {
```
Comment (herrmannplatz) [inline test/unit_tests/team/TeamRepositorySpec.js]: what about this test? do we need it?

File: app/script/user/UserRepository.js [hunk 3]
```diff
@@ -946,4 +957,17 @@ z.user.UserRepository = class UserRepository {
...
+    user_ets.forEach((user_et) => {
+      console.log('user_et', user_et);
```
Comment (bennycode) [inline app/script/user/UserRepository.js]: console.log

File: app/script/user/UserRepository.js [hunk 4]
```diff
@@ -946,4 +957,17 @@ z.user.UserRepository = class UserRepository {
...
+      const is_team_member = !!team_members.find((member) => member.id === user_et.id);
+      console.log('is_team_member', is_team_member);
```
Comment (bennycode) [inline app/script/user/UserRepository.js]: console.log

**Pull request merged**